assets/properties-panel.css

```diff
@@ -446,6 +446,21 @@
   text-decoration: underline;
 }
 
+.bio-properties-panel-feelers-editor.bio-properties-panel-input {
+  padding: 0;
+}
+
+.bio-properties-panel-feelers-input .cm-editor
+{
+  min-height: 32px;
+  max-height: 215px;
```

Comment:
Why do we need restrictions here and not for the "normal" FEEL editor?

Reply:
It's a design decision that was taken during a little UX session. In any case since we might merge the two editors, I think let's not worry about how the other one displays for now.